src/Components/NavBar/Menus/NavBarNewNotifications.tsx

```diff
@@ -0,0 +1,22 @@
+import { Flex } from "@artsy/palette"
+import { Notifications } from "Components/Notifications/Notifications"
+import { useScrollLock } from "Utils/Hooks/useScrollLock"
+
+export const NavBarNewNotifications = () => {
+  const { lockScroll, unlockScroll } = useScrollLock()
+
+  return (
+    <Flex
+      width={420}
+      p={2}
+      maxHeight="90vh"
+      overflowY="scroll"
+      flexDirection="column"
+      aria-live="assertive"
+      onMouseEnter={lockScroll}
+      onMouseLeave={unlockScroll}
```

Comment:
Disable scroll for parent containers when dropdown menu is focused

#### Without disable scroll for parent

#### With disable scroll for parent
https://user-images.githubusercontent.com/3513494/187617627-d327e321-d6c6-4b7b-9225-bd1a13091a24.mp4